public/app/plugins/panel/debug/plugin.json

```diff
@@ -2,6 +2,7 @@
   "type": "panel",
   "name": "Debug",
   "id": "debug",
+  "alias": "debugX",
```

Comment:
I see this plugin changing the id again to `grafana-debug-panel` so we may need to support a string array for alias(es)